config_example.json

```diff
@@ -4,10 +4,24 @@
  "globalHTTPTimeout": 15000000000,
  "logging": {
   "enabled": true,
-  "file": "debug.txt",
-  "colour": false,
-  "level": "DEBUG|WARN|INFO|ERROR|FATAL",
-  "rotate": false
+  "level": "INFO|WARN|DEBUG|ERROR",
+  "output": "console",
+  "file": {
```

Comment:
fileSettings for the JSON key name

Reply:
whoops missed that when i made the change last night! updated